source/administration/devops-command-center.rst

```diff
@@ -152,6 +152,8 @@ Starting an incident opens an interactive dialog. Provide an incident name and s
 
 The creator of an incident is automatically added as the first member and becomes the commander.
 
+When the incident is created, it is announced in the configured channel. If the channel does not exist or is archived/deleted, the bot posts a message in the channel warning the user about it.
```

Comment:
What's the use case for a channel not existing and being announced? Would it be if a user tries to access it? If so, how would they access it if doesn't exist? Or is it if the channel did exist and was archived/deleted only?

Reply:
This is an edge case that may not need docs, actually. The case is: a user configures channel A as the announcement channel in the playbook. For some external reason, channel A is later archived/deleted, but the playbook is not updated with another channel. Then, an incident is created with that playbook, and the system tries to announce the incident creation in channel A: as channel A no longer exists, the bot posts a message in the channel warning the user that the announcement could not be made.